packages/rum-core/src/performance-monitoring/performance-monitoring.js

```diff
@@ -258,6 +260,18 @@ export default class PerformanceMonitoring {
     const configService = this._configService
     const transactionService = this._transactionService
 
+    // do not process calls to our own endpoints
+    if (task.data && task.data.url) {
```

Comment:
If the request's url points to our own endpoints we will NOT process it. This new behaviour works for XHR and fetch. 

P.S. Nevertheless, I'm not removing the XHR_IGNORE logic since it might be useful for other uses cases (although I would advocate to remove it in future PRs if there are not valid use cases)

P.S.2. This new logic also helps to cover the case where our users make use of fetch polyfills that rely on XHR

Reply:
Can you make an issue and get rid of XHR_IGNORE in the next PR's? I don't think its required as we have this logic which can handle both now.

Reply:
Created! https://github.com/elastic/apm-agent-rum-js/issues/1164